README.md

```diff
@@ -1,4 +1,4 @@
-# <img src="https://raw.githubusercontent.com/jpmorganchase/quorum/master/logo.png" width="200" height="200"/>
+# <img src="https://raw.githubusercontent.com/jpmorganchase/quorum/master/logo.png" width="200" height="35"/>
```

Comment:
we should also update the URL to `https://raw.githubusercontent.com/ConsenSys/quorum/master/logo.png` (aka replace `jpmorganchase` with `ConsenSys`)

Reply:
done